custom/succeed/expressions.py

```diff
@@ -0,0 +1,32 @@
+from casexml.apps.case.models import CommCareCase
+from corehq.apps.userreports.specs import TypeProperty
+from dimagi.ext.jsonobject import JsonObject
+
+
+class ReferencedIdExpressionSpec(JsonObject):
+    type = TypeProperty('succeed_referenced_id')
+
+    def __call__(self, item, context=None):
+        referenced_id = item['indices'][0]['referenced_id']
+        return referenced_id
+
+
+class FullNameExpressionSpec(JsonObject):
```

Comment:
I think you can just have just used a `related_doc` expression with the `succeed_referenced_id` as the id lookup and then a normal property lookup as the related doc expression, right? would be nice not to add this unless it's necessary.